### PR TITLE
[DBZ-1405] Update Cassandra Connector Documentation

### PR DESCRIPTION
**Pull request by bingqinzhou**

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/cassandra.adoc [hunk 1]
```diff
@@ -1013,3 +1008,13 @@ kafka.consumer.ssl.key.password=test1234
 ----
 
 Be sure to consult the http://kafka.apache.org/documentation.html[Kafka documentation] for all of the configuration properties for Kafka producers.
+
+The connect supports the following kafka connect converters for key/value serialization:
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/cassandra.adoc]:
```suggestion
The connector supports the following Kafka Connect converters for key/value serialization:
```

File: documentation/modules/ROOT/pages/connectors/cassandra.adoc [hunk 2]
```diff
@@ -770,38 +770,33 @@ The Cassandra connector should be deployed each Cassandra node in a Cassandra cl
 [[example-configuration]]
 === Example configuration
 
-The following represents an example .properties configuration file for Cassandra Connector:
+The following represents an example .properties configuration file for running and testing Cassandra Connector locally:
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/cassandra.adoc]:
```suggestion
The following represents an example .properties configuration file for running and testing the Cassandra Connector locally:
```

**Comment by bingqinzhou** [comment]
> Two suggestions inline; and shouldn't the new `Duration` mapping be reflected in this change somehow, too?

@gunnarmorling Thank you and I've committed your inline suggestions. I also added mapping for `Duration` to `io.debezium.time.MicroDuration`, will update this accordingly when PR #1029 is merged and released :)

**Comment by gunnarmorling** [comment]
Thanks, @bingqinzhou. I've squashed everything into one commit and merged it to the _upstream/development_docs_ branch. This branch contains all doc changes that should be published to the website after the next release. I've also updated the duration mapping description as per the nano-seconds change.

**Pull request closed without merging**